### PR TITLE
Attempt to fix code-ql issues

### PR DESCRIPTION
**Pull request by martintmk**
## The issue or feature being addressed

#980 

## Details on the issue fix or feature implementation

Try to stabilize `code-ql` build by ignoring our analyzers.

## Confirm the following

- [ ]  I started this PR by branching from the head of the default branch
- [ ]  I have targeted the PR to merge into the default branch
- [ ]  I have included unit tests for the issue/feature
- [ ]  I have successfully run a local build

### CODE REVIEW

File: eng/Analyzers.targets [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <Project>
-  <ItemGroup Label="Analyzers">
+  <ItemGroup Label="Analyzers" Condition="$(SKIP_POLLY_ANALYZERS) == ''">
```
Comment (martincostello) [inline eng/Analyzers.targets:2]:
```suggestion
  <ItemGroup Label="Analyzers" Condition=" '$(SKIP_POLLY_ANALYZERS)' == '' ">
```

File: eng/Analyzers.targets [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <EditorConfigFiles Include="$(MsBuildThisFileDirectory)analyzers\$(ProjectType).globalconfig" />
   </ItemGroup>
 
-  <PropertyGroup>
+  <PropertyGroup Condition="$(SKIP_POLLY_ANALYZERS) == ''">
```
Comment (martincostello) [inline eng/Analyzers.targets:12]:
```suggestion
  <PropertyGroup Condition=" '$(SKIP_POLLY_ANALYZERS)' == '' ">
```

**Review by martincostello — APPROVED** [review]
If it works 👍

**Comment by martintmk** [comment]
> If it works 👍

WORKS! :)

**Pull request merged**